server/api/adapters/redis.ts

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config()
+const redis = require('redis'),
+redist_options = {
+	"host" : process.env.REDIS_HOST,
+	"port" : 6379
+},
+client = redis.createClient(redist_options)
+
+client.on('connect', () => {
+    console.log('Redis client connected');
+});
+
+client.on('error', (err: any) => {
+    console.log('Something went wrong ' + err);
+});
+
+module.exports.setVal = (key: string, val: string): string => client.set(key, val);
```

Comment:
should be promisified. e.g. 
```
const {promisify} = require('util');
const setAsync = promisify(client.set).bind(client);
```

Reply:
this is not done using promise, because i dont expect any value from it, 

```
client.set() returns "OK"
```